### PR TITLE
Added Exponential Search in Python

### PR DESCRIPTION
**Pull request by JayeshGhosh**
Adding Exponential Search Algorithm using Python

### CODE REVIEW

**Comment by Zircoz** [comment]
@JayeshGhosh Can you also add a bit of documentation for it?

**Comment by JayeshGhosh** [comment]
Added short Explaination as comments in file

File: exponentialsearch.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+'''
+Explanation:
```
Comment (Zircoz) [inline exponentialsearch.py:2]: Fix the spelling typo

File: exponentialsearch.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            return binarySearch(data_list,mid + 1,high,value)
+    if(high<low):
+        return -1
+
+#binary search logic from driver code
+index=binarySearch(list1,i/2,min(i,len(list1)),val)
+if(index==-1):
+    print("Element not found")
+else:
+    print("Element found at ",index)
```
Comment (Zircoz) [inline exponentialsearch.py]: you don't need white space after "at" because you are using comma (and not concatenation)

**Review by Zircoz — CHANGES_REQUESTED** [review]

**Review by Zircoz — APPROVED** [review]

**Comment by JayeshGhosh** [comment]
Thanks for suggestions 🙂

**Pull request merged**